### ISSUE TITLE
Docker image for ARM (rpi) available now! Use PW2PY within minutes :)

### ISSUE THREAD

**Issue by gruijter** [SevenW/Plugwise-2-py#77]
I have created a docker image that can run on a raspberry pi. It reduces the installation effort drastically to only a few minutes.

See instructions here:
https://hub.docker.com/r/gruijter/plugwise-2-py

The image is based on the Plugwise-2-py V2.0.

@SevenW maybe you can add this link to the installation instructions?

@ Anybody: Please let me know if it works for you, or any other feedback at my github repo https://github.com/gruijter/PW2PY_DOCKER

**Comment by gruijter** [comment]
I now also created a full rpi4 SD card image. See info here: https://github.com/gruijter/PW2PY_DOCKER/tree/main/rpi4img

**Comment by SevenW** [comment]
I also created a Docker, on synology, back in October. Somehow I did not push that fro local to github. I need to check whicjh I like better. yours or mine :).

**Comment by gruijter** [comment]
I only tested mine on a rpi. Why not have both? One for rpi, and one for synology :)

**Comment by gruijter** [comment]
So, did you try my docker? I made a 64bit version as well.

**Comment by sjoerdos92** [comment]
@SevenW Can you share your docker file?

**Comment by gruijter** [comment]
Hi Sjoerd. I posted the link to dockerhub and github here already. What more is it you are looking for?

**Comment by sjoerdos92** [comment]
I couldn't get your image to work on my docker. I'm running on x86, not ARM. Yours is only for ARM, correct?

**Comment by gruijter** [comment]
Correct. But maybe you can reuse my dockerfile to build an x86 version. You can find it on my github link.
